client/app/templates/application.hbs

```diff
@@ -5,15 +5,26 @@
   as |banner|
 >
   <banner.title>
-    <LinkTo @route="index" class="site-title">
+    <LinkTo @route="application" class="site-title">
       Applicants
       <small class="site-subtitle show-for-medium">File and track your applications</small>
     </LinkTo>
   </banner.title>
   <banner.nav>
     <ul class="menu vertical medium-horizontal">
-      <li><a href="https://zap.planning.nyc.gov"><span class="text-a11y-orange">Find Projects</span></a></li>
-      <li><a href=""><FaIcon @icon="user-circle" @prefix="fas" class="text-light-gray tiny-margin-right" /> <span class="text-a11y-orange">Sign In</span></a></li>
+      <li><a href="https://zap.planning.nyc.gov">Find Projects</a></li>
+      <li>
+        {{#if this.session.isAuthenticated}}
+          <button {{on "click" this.invalidateSession}}
+            href="#"
+            type="button"
+          >
+            Logout
+          </button>
+        {{else}}
+          <SignInButton />
```

Comment:
@allthesignals I think the layout in the sign-in button component needs to be adjusted:
![image](https://user-images.githubusercontent.com/5316367/78608735-687c6400-782f-11ea-8f50-40e764fb307b.png)

Reply:
bah crap. it needs to be a button element bc of accessibbility. can't be anchor.

Reply:
@andycochran need some help with this — this is styled for anchor tags, but we need to use button elements bc you can't bind actions to anchor elements. thoughts?

I'm going to move into new issue so we can move on